netlify.toml

```diff
@@ -0,0 +1,5 @@
+[template.environment]
+  WHITELISTED_ADDRESSES=0x692a70D2e424a56D2C6C27aA97D1a86395877b3A 0xa88c953d9bFcdcFa46d9ec29586820D55C4EC0dC
+  WHITELISTED_METHODS=vote(uint8) test(address,uint8)
```

Comment:
When you have spaces in env variables, don't you need " around them?

Reply:
It's been working for me without the quotes (Locally at least). So I'd say you don't need it